applications/ConvectionDiffusionApplication/python_scripts/convection_diffusion_explicit_solver.py

```diff
@@ -0,0 +1,115 @@
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# Import applications
+import KratosMultiphysics.ConvectionDiffusionApplication as ConvectionDiffusionApplication
+
+# Import base class file
+from KratosMultiphysics.ConvectionDiffusionApplication import (
+    convection_diffusion_base_solver,
+)
+
+
+def CreateSolver(model, custom_settings):
+    return ConvectionDiffusionExplicitSolver(model, custom_settings)
+
+
+class ConvectionDiffusionExplicitSolver(
+    convection_diffusion_base_solver.ConvectionDiffusionBaseSolver
+):
+    """
+    The explicit class for convection-diffusion solvers.
+    See convection_diffusion_base_solver.py for more information.
+    """
+
+    def __init__(self, model, custom_settings):
+        # Construct the base solver and validate the remaining settings in the base class
+        super(ConvectionDiffusionExplicitSolver, self).__init__(model, custom_settings)
+
+        # Overwrite the base solver minimum buffer size
+        self.min_buffer_size = 2
+
+        element_name = self.settings["element_replace_settings"][
+            "element_name"
+        ].GetString()
+        if self.settings["use_orthogonal_subscales"].GetBool() is True:
+            if element_name in (
+                "QSConvectionDiffusionExplicit",
+                "DConvectionDiffusionExplicit",
+            ):
+                self.main_model_part.ProcessInfo.SetValue(
+                    KratosMultiphysics.OSS_SWITCH, 1
+                )
+            else:
+                err_msg = self.__class__.__name__
+                err_msg += (
+                    "The selected element",
+                    element_name,
+                    "does not support OSS projection. Select QSConvectionDiffusionExplicit or DConvectionDiffusionExplicit instead.",
+                )
+                raise Exception(err_msg)
+        else:
+            if element_name in (
+                "QSConvectionDiffusionExplicit",
+                "DConvectionDiffusionExplicit",
+            ):
+                self.main_model_part.ProcessInfo.SetValue(
+                    KratosMultiphysics.OSS_SWITCH, 0
+                )
+            else:
+                err_msg = self.__class__.__name__
+                err_msg += (
+                    "The selected element",
+                    element_name,
+                    "does not support OSS projection. Select QSConvectionDiffusionExplicit or DConvectionDiffusionExplicit instead.",
+                )
+                raise Exception(err_msg)
+
+        KratosMultiphysics.Logger.PrintInfo(
+            self.__class__.__name__, "Construction finished"
+        )
+
+    @classmethod
+    def GetDefaultParameters(cls):
+        default_settings = KratosMultiphysics.Parameters(
+        """
+        {
+            "use_orthogonal_subscales" : false,
+            "explicit_parameters" : {
```

Comment:
Why an extra level for a single field? Furthermore, I think that `use_orthogonal_subscales` and `dynamic_tau` must be at the same level.

If you want to encapsulate all the formulation-related settings we can create an extra level called `formulation` as we did in the fluid solvers.However this needs to be done in a separate PR as it would affect the entire application and the GUI.

Reply:
Removed extra level.